OCP/Example3.md

```diff
@@ -0,0 +1,23 @@
+### Usage
+
+---
+#### Where
+
+https://github.com/angular/angular/blob/main/packages/compiler-cli/integrationtest/src/jit_summaries.ts#L27
```

Comment:
➕ ➖
I'm not really sure it's about OCP. I didn't get the meaning of these classes.